doc/themes/the-things-stack/assets/js/theme.js

```diff
@@ -24,4 +24,65 @@ document.addEventListener('DOMContentLoaded', function () {
 			});
 		});
 	}
+
+  const tabs = [...document.querySelectorAll('#tabs li')];
+  const tab_content = [...document.querySelectorAll('#tab-content section')];
+  const active_class = 'is-active';
```

Comment:
```suggestion
  const tabContent = [...document.querySelectorAll('#tab-content section')];
  const activeClass = 'is-active';
```

Nit, if you want to be more in line with frontend code style.